README.md

```diff
@@ -27,6 +27,17 @@ var map = L.map('map', {
 L.tileLayer.iiif('http://example.com/iiifimage.jp2/info.json').addTo(map);
 ```
 
+### With error handler option (Passing in your own custom function, as below)
+
+```
+function errorHandler() {
+  $('#viewer').removeClass("not_loaded").height("30px")
+     .html("<div class='error'>Sorry, an error occurred while loading the image.</div>");
+}
+
+L.tileLayer.iiif('http://example.com/iiifimage.jp2/info.json', {errorHandler: errorHandler}).addTo(map);
```

Comment:
Can you add this option to the `options` table? Also it might be good to specify that this only handles errors from a null response.